week-2/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,17 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+  if (mark >= 80 && mark <=100 ) {
+    return "A";
+  } else if (mark <=80 && mark >=60 ) {
+    return "B";
+  }else if (mark <=60 && mark >=50 ) {
+    return "C";
+  } else {
+    return "F"
+  }
```

Comment:
Nearly perfect. There's a small issue though which is that the first `if` triggers on 80, so the second won't get to match it, and the second `if` triggers on 60, so the third won't get to match it.
The instructions say "if the mark is greater than 60 then the grade is "B", if the mark is 60 or lower the grade is "C"" - so what grade should 60 actually be? And what grade does your code mark for 60?